int/hwsc-user-svc/hwsc-user-svc.js

```diff
@@ -1,18 +1,15 @@
 const grpc = require('grpc');
 const protoLoader = require('@grpc/proto-loader');
 
-const HWSC_USER_PROTO_PATH = '../lib/';
-const HWSC_USER_SVC_PROTO_PATH = `${__dirname}/proto/`;
+const PROTO_PATH = `${__dirname}/../..`;
```

Comment:
Set the path relative to all the protocol buffers.
`path = github.com/hwsc-org/hwsc-api-blocks`